### ISSUE TITLE
A Question about Data

### ISSUE THREAD

**Issue by zhouhao-learning** [matterport/Mask_RCNN#1165]
I want to know, if I do image target location detection, then my training data must be labeled one by one? For example, I want to detect the people and cars in the image, so when I train, do I have to label the people and cars in the training data?

**Comment by stygian2a** [comment]
Yes of course!
You might find already labeled data online though

**Comment by zhouhao-learning** [comment]
@stygian2a 
Can there be any algorithm to replace this annotation? Because if it's a cell image, it's too hard to label one cell at a time.

**Comment by stygian2a** [comment]
No, I'm sorry. There are some software that helps you make manual annotations though. Can't you find an already annotated dataset?

Besides, if you had an algorithm that automatically  labelled your images, why would be the use of the training? You would already have a program that detect whatever you are trying to  detect !

**Comment by HAMZARaouia** [comment]
The are some tools for manual labeling like [Labelme](https://github.com/wkentaro/labelme) if you need the semantic segmentation otherwise you can use [LabelImg](https://github.com/tzutalin/labelImg) to do only detection
